scripts/utils/log.js

```diff
@@ -0,0 +1,16 @@
+const chalk = require('chalk');
+
+const TAG_COLOR = 'cyan';
+const info = function(tag, ...args) {
+  console.log(chalk[TAG_COLOR](tag), ...args);
+};
+
+const error = function(tag, ...args) {
+  console.log(chalk[TAG_COLOR](tag), chalk.red(...args));
+};
+
+const warn = function(tag, ...args) {
+  console.log(chalk[TAG_COLOR](tag), chalk.yellow(...args));
+};
+
+module.exports = { info, error, warn };
```

Comment:
this is a bit over engineered yeah? We have only 2-3 log statements, I think console.log is ok.

Reply:
We can remove this if the final build process is lean.

Reply:
not sure what you mean by that? is this not ready for review? what's left to be done?

Reply:
> is this not ready for review
I thought you had your doubts with Rollup. Thought you would like to see both current and rollup in action together and maybe compare the builds. This is PR is intended only to compare the two.

Sorry if this is add too much cognitive load. I thought a discussion was necessary before we settled with rollup

Reply:
I'm really sorry :) I just misunderstood what was happening! I'm fine with rollup, the existing build is really just the minimal i do for every project to bootstrap it. I'm more than happy to mimic the react build of using rollup for a flat common js bundle 👍